Grammar/python.gram

```diff
@@ -642,7 +642,10 @@ invalid_assignment:
     | a=expression ':' expression ['=' annotated_rhs] {
         RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "illegal target for annotation") }
     | a=expression ('=' | augassign) (yield_expr | star_expressions) {
```

Comment:
This was not changed in this PR, but I now realize that it should probably be like this:
```suggestion
    | a=star_expressions ('=' | augassign) (yield_expr | star_expressions) {
```

Without this we get:
```
➜  cpython git:(pr/20076) ./python               
Python 3.9.0a6+ (heads/pr/20076:127ce7376c, May 14 2020, 01:59:27) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 1, 2 = 1
  File "<stdin>", line 1
    1, 2 = 1
         ^
SyntaxError: invalid syntax
```